docs/guide.md

```diff
@@ -0,0 +1,764 @@
+# Guide to Running E3SM
+
+## About This Guide
+
+This guide is intended to walk users step-by-step through the simulation process -- from setting up a simulation to post-processing, long-term archiving, and publishing it.
+
+## Useful Aliases
```

Comment:
I'd like this moved to later in the document.

Reply:
The aliases are specifically used in two places in the guide though. Those references then won't make sense unless people have hopped down to the aliases appendix. I suppose the specific references could be swapped out with something like "check on your batch jobs (see alias section)", which I think is what you're suggesting here.

Reply:
Using the hierarchy of guides, we could put things like aliases in an ancillary/appendix section and link to it when aliases are used.